src/pages/limyujin/Main/Components/Feeds.js

```diff
@@ -0,0 +1,201 @@
+            <ul className="feedYJ__feature-btn-box">
+              <li className="feedYJ__feature-btn-list">
+                <button className="feedYJ__feature-btn" onClick={handleLike}>
+                  <i
+                    className={`${
+                      isFeedLikePressed ? 'fas fa-heart' : 'far fa-heart'
+                    }`}
+                  ></i>
+                </button>
+              </li>
+              <li className="feedYJ__feature-btn-list">
+                <button className="feedYJ__feature-btn">
+                  <FontAwesomeIcon icon={faComment} className="comment" />
+                </button>
+              </li>
+              <li className="feedYJ__feature-btn-list">
+                <button className="feedYJ__feature-btn">
+                  <FontAwesomeIcon icon={faPaperPlane} className="comment" />
+                </button>
+              </li>
+            </ul>
+            <ul className="feedYJ__feature-btn-box">
+              <li className="feedYJ__feature-btn-list">
+                <button className="feedYJ__feature-btn">
+                  <FontAwesomeIcon icon={faBookmark} className="bookmark" />
+                </button>
+              </li>
+            </ul>
+          </section>
+          <section className="feedYJ__texts">
+            <div className="feedYJ__texts-column feedYJ__likes">
+              <span>좋아요 </span>
+              <span className="js-likes-count">
+                {isFeedLikePressed
+                  ? (like + 1).toLocaleString()
+                  : like.toLocaleString()}
+              </span>
+              <span>개</span>
+            </div>
+            <div className="feedYJ__texts-column feedYJ__summary">
+              <span className="feedYJ__summary-id">{author}</span>
+              <span>{text}</span>
+              <span>...</span>
+              <button className="feedYJ__summary-btn">더 보기</button>
+            </div>
+            <div className="feedYJ__texts-column js-feed-comments">
+              {commentTexts.map(comment => (
+                <CommentColumn
+                  commentTexts={comment.text}
+                  key={comment.id}
+                  id={comment.id}
+                  handleDelete={handleDelete}
+                />
+              ))}
+            </div>
+            <div className="feedYJ__time feedYJ__texts-column">
+              <span>{getTime(time)}</span>
+            </div>
+          </section>
+          <form name="comment-form" className="feedYJ__form">
+            <FontAwesomeIcon icon={faSmile} className="smile" />
+            <input
+              type="text"
+              placeholder="댓글달기..."
+              value={commentValue}
+              className="feedYJ__input"
+              onChange={handleInput}
+            />
+            <button
+              className={`feedYJ__submit-btn ${
+                commentValue.length > 0 ? '' : 'feedYJ__submit-btn--opacity'
```

Comment:
```suggestion
                !commentValue.length > 0 && 'feedYJ__submit-btn--opacity'
```
값이 true거나 false일때만 적용시킬 값이 있다면 굳이 삼항연산자를 사용안하고 `&&` or `||` 를 사용하실 수 있습니다.

Reply:
&& 트릭, || 트릭 저번부터 계속 알려주셨는데 제가 자꾸 쓸 생각을 못하네요 ㅠㅠ 수정 완료했습니다 ! 기능도 의도대로 잘 됩니다 😆